docs/kubebuilder_v0_v1_difference.md

```diff
@@ -0,0 +1,124 @@
+# Kubebuilder v0 v.s. v1
+
+## command difference
+  - kubebuilder v0 has `init`, `create controller`, `create resouece`, `create config`, `generate` commands and the workflow is
+
+```
+  kubebuilder init --domain example.com
+  kubebuilder create resource --group <group> --version <version> --kind <Kind>
+  GOBIN=${PWD}/bin go install ${PWD#$GOPATH/src/}/cmd/controller-manager
+  bin/controller-manager --kubeconfig ~/.kube/config
+
+  kubectl apply -f hack/sample/<resource>.yaml
+  docker build -f Dockerfile.controller . -t <image:tag>
+  docker push <image:tag>
+  kubebuilder create config --controller-image <image:tag> --name <project-name>
+  kubectl apply -f hack/install.yaml
+```
+  Everytime the resource or controller is updated, users need to run `kubebuilder generate` to regenerate the project.
+  - kubebuilder v1 has `init`, `create api` commands and the workflow is
+  
+```
+  kubebuilder init --domain example.com --license apache2 --owner "The Kubernetes authors"
+  kubebuilder create api --group ship --version v1beta1 --kind Frigate
```

Comment:
thinking more about it...wondering if we can make `resource/api` same way under the hood for new projects, then we all the commands will be backward compatible.

Reply:
I don't think we need to change it to `create resource`. In v1 command `kubebuilder create api`, there are two flags `--resource` and `--controller`. Changing to `create resource` may cause confusion for `create resource --controller`.

Reply:
Ack.